### ISSUE TITLE
Add sort by

### ISSUE THREAD

**Issue by el1t** [elmzteam/bydesign#21]
Most viewed, comments, oldest...
Some features may overlap/be desired with search functionality

**Comment by zwade** [comment]
Would this be server side or client side?

My instinct is that this sort of thing is client

**Comment by bluepichu** [comment]
That'd necessarily have a server component, unless you want to send the client the entire post list...

**Comment by zwade** [comment]
Yeah, you would have to. The thing is, for any kind of advanced sorting, doing it server side balloons really quickly, similar to searching.
